tests/unit-core/test-api.c

```diff
@@ -1130,6 +1130,50 @@ main (void)
 
   jerry_release_value (args[1]);
 
+  {
+    /*json parser check*/
+    jerry_value_t parsed_json;
+    jerry_value_t parsed_data;
+    char data_check[]="John";
+    jerry_value_t key = jerry_create_string ((const jerry_char_t *) "name");
+    const char *data = "{\"name\": \"John\", \"age\": 5}";
+    parsed_json = jerry_json_parse (data);
+    jerry_value_t has_prop_js = jerry_has_property (parsed_json, key);
+    bool has_prop = jerry_get_boolean_value (has_prop_js);
+    TEST_ASSERT (has_prop == true);
+    jerry_release_value (has_prop_js);
+    parsed_data = jerry_get_property (parsed_json, key);
+    TEST_ASSERT (jerry_value_is_string (parsed_data)== true);
+    jerry_size_t buff_size = (jerry_size_t) jerry_get_string_length (parsed_data);
+    char buff[jerry_get_string_length (parsed_data)];
+    jerry_char_t *buff_p = (jerry_char_t *) buff;
+    jerry_string_to_char_buffer (parsed_data, buff_p, buff_size);
+    buff[buff_size] = '\0';
+    TEST_ASSERT (strcmp ((const char *) data_check , (const char *) buff)  == false);
```

Comment:
I do not think it is necessary to create a variable for the data_check content, just put the string here.